### DISCUSSION TITLE
make fatal errors

### DISCUSSION THREAD

**Discussion by helaozhao** [w3c/webcodecs#544] [Q&A] [answered]
Hi there, when I make from master branch, I get the following hint:
"LINE 172: W3C policy requires Privacy Considerations and Security Considerations to be separate sections, but you appear to have them combined into one."

Can anybody help please?

**Comment by padenot** [comment, marked as answer]
Thanks for the heads-up, I wrote a fix in https://github.com/w3c/webcodecs/pull/545.

**Reply by hehe1112** [reply]
quick fix. Great!

**Comment by dalecurtis** [comment]
Errors are fatal, so closing as fixed.
